frontend/src/api/pub.js

```diff
@@ -3,9 +3,11 @@ import { baseURL } from "@/utils/constants";
 
 export async function fetch(url, password = "") {
   url = removePrefix(url);
+  let deiso = new TextDecoder("iso88591");
```

Comment:
On the server side, we support uft8 only. We can't use `iso88591` for that

Reply:
I think we can try to use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent and decode it on the server side